src/audit/audit.go

```diff
@@ -102,7 +103,7 @@ func (a *Audit) Init() error {
 }
 
 // LogReadEvent used to handle the read-only event.
-func (a *Audit) LogReadEvent(t string, user string, host string, threadID uint32, query string, affected uint64, startTime time.Time) {
+func (a *Audit) LogReadEvent(t string, user string, host string, threadID uint32, query string, status uint32, affected uint64, startTime time.Time) {
```

Comment:
I think we can use a struck instead of  an arg list in LogReadEvent(...) func.

Reply:
proxy/audit.go auditLog function called. The args will write in event.

Reply:
> I think we can use a struck instead of an arg list in LogReadEvent(...) func.

The struct is `event`. `LogReadEvent`  is called by  `spanner.auditLog` outside the `audit`.

Reply:
> I think we can use a struck instead of an arg list in LogReadEvent(...) func.

Arg list is better here, by this we sure to know which events you should to log.

Reply:
> > I think we can use a struck instead of an arg list in LogReadEvent(...) func.
> 
> Arg list is better here, by this we sure to know which events you should to log.

Got!

Reply:
> > I think we can use a struck instead of an arg list in LogReadEvent(...) func.
> 
> The struct is `event`. `LogReadEvent` is called by `spanner.auditLog` outside the `audit`.

Make sense.